knowledge/insects/worker_bees/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: ishaksyed2
+domain: insect
+seed_examples:
+- answer: | The working bee normally does not have reproductive capacity
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!